resources/views/templates/watch/header.blade.php

```diff
@@ -0,0 +1,129 @@
+    {{ Html::style(asset('css/frontend/slick.css')) }}
+    {{ Html::style(asset('css/frontend/slick-theme.css')) }}
+    {{ Html::style(asset('css/frontend/nouislider.min.css')) }}
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.1/css/all.css" integrity="sha384-gfdkjb5BdAXd+lj+gudLWI+BXq4IuLW5IT+brZEZsLFm++aCMlF1V92rMkPaX4PP"
+        crossorigin="anonymous">
+    {{ Html::style(asset('css/frontend/style.css')) }}
+
+</head>
+
+<body>
+    <header>
+        <div id="top-header">
+            <div class="container">
+                <ul class="header-links pull-left">
+                    <li><a href="#"><i class="fa fa-phone"></i>@lang('messages.phone')</a></li>
+                    <li><a href="#"><i class="fas fa-envelope"></i> @lang('messages.email')</a></li>
+                    <li><a href="#"><i class="fa fa-map-marker"></i>@lang('messages.address')</a></li>
+                </ul>
+                <ul class="header-links pull-right">
+                    @auth
+                    <li><a href="#"><i class="fas fa-user"></i>@lang('messages.adminpage')</a></li>
+                    <li class="dropdown"><a href="#">{{ Auth::user()->name }}</a>
+                        <div class="dropdown-content bg-dark">
+                            <a>
+                                @lang('messages.logout')
+                            </a>
+
+                            {!! Form::open(['route' => 'logout', 'method' => 'POST', 'id' => 'logout-form',
+                                'style' => 'display: none']) !!}
+                            {!! Form::close() !!}
+                        </div>
+                    </li>
+                    @else
+                    <li><a href="#"><i class="fas fa-user"></i>@lang('messages.login')</a></li>
+                    <li><a href="#"><i class="fas fa-user"></i>@lang('messages.register')</a></li>
+                    @endauth
+                </ul>
+            </div>
+        </div>
+        <div id="header">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-3">
+                        <div class="header-logo">
+                            <a href="#" class="logo">
+                                <img src="{{ asset('frontend/img/logo.png') }}" alt="">
+                            </a>
+                        </div>
+                    </div>
+                    <div class="col-md-6">
+                        <div class="header-search">
+
+                            {!! Form::open() !!}
+                                <select class="input-select">
+                                    <option value="0">@lang('allcategory')</option>
+                                        @for ($i = 0; $i < count($menus); $i++) 
+                                            @if ($menus[$i]->parent_id===0)
+                                                <option value="1">{!! $menus[$i]->name !!}</o>
```

Comment:
tại sao value = 1

Reply:
[
                'name' => 'Laptop 1',
                'picture' => './frontend/img/shop01.png',
                'parent_id' => 1,
            ],
            [
                'name' => 'Laptop 2',
                'picture' => './frontend/img/shop01.png',
                'parent_id' => 1,
            ],
có nghĩa là trong thể loại laptop ===0 thì có các loại laptop nhỏ e đặt ===1

Reply:
nó sẽ tìm thằng nào có parrent_id=1 thì nó in ra.

Reply:
vậy nếu có dòng laptop nhỏ hơn thì sao? 
ví dụ: con của laptop 0 là 1, con của laptop 1 là 2, ... 
vậy ở đây làm như này có hợp lý không?